include/utility/vpd_specific_utility.hpp

```diff
@@ -377,5 +377,30 @@ inline void getVpdDataInVector(const std::string& vpdFilePath,
         throw;
     }
 }
+
+/**
+ * @brief API to get inventory path without prefix.
+ *
+ * API finds if the given inventory object path is prefixed with
+ * "/xyz/openbmc_project/inventory" and returns the object path by removing the
+ * prefix if present.
+ *
+ * @param[in] i_invObjPath - Inventory object path.
+ *
+ * @return Unprefixed inventory object path.
+ */
+inline std::string getUnprefixedInventoryPath(const std::string& i_invObjPath)
```

Comment:
We don;t need thisA API now.
This is a known defect where we are calling PIM with complete path and this fix will be wrapped in call PIM API itself, so that user can be abstracted from these kind of addition/removal operation.

For now your code can update the full path.